docs/add-zoom-licenses.md

```diff
@@ -0,0 +1,19 @@
+Prerequisite: You must be a Zoom administrator
+
+1) Login to zoom and navigate to the **Users Management** tab
+
+<img src="img/zoom-users.png" width=512>
+
+2) Select the **Purchase more licenses** link.
+
+3) Select **Edit Plan**
+
+4) Increment the **Number of users** to the desired (increased) number of users.
+
+5) Enter the requisite information that Zoom prompts you to provide.  When asked for the email address, please use the precedent used for all of the other accounts (i.e. logins+slash-z-command-{NUMBER}@hackclub.com)
+
+6) You will eventually be prompted to enter in the CVV of the credit card associated with the Zoom administrator account.  Reach out to graham@hackclub.com or msw@hackclub.com for that information.
```

Comment:
My address is max@hackclub.com-- msw@ isn't monitored!

Reply:
Whoops! willfix